variables.tf

```diff
@@ -85,3 +85,11 @@ variable "allow_deletion" {
 
   description = "Set this to true in order to allow destroying stateful resources (buckets, DBs, etc.)"
 }
+
+variable "cromwell_version" {
+  type     = number
```

Comment:
Cromwell versions are _usually_ numbers, but occasionally they have [hotfix releases](https://github.com/broadinstitute/cromwell/releases/tag/68_hotfix_8e12ab5) that aren't numbers. I'd recommend making this input a string.